readme.md

```diff
@@ -99,9 +99,13 @@ Then run `rake double_doc`, which will generate a `readme.md` in the root of the
 If a gh-pages branch exists, run `rake doc:publish` to generate html documentation and push it to your github pages.
 
 ### Notes
- - Tested on ruby 2.0+
+ - Tested on ruby 3.0+
  - Does not work on jruby because of its dependency on redcarpet.
 
+### Release
+
+After merging your changes, create a PR with a version bump and updated changelog. After that PR gets merged, create a new tag (by running `gem_push=no rake release` or via Github releases). This will trigger the publishing workflow (which will need to be [approved in Github Actions](https://github.com/zendesk/double_doc/actions/workflows/publish.yml)).
```

Comment:
why do we have 2 readmes with the same content ? ... can we just put this into the main one ?

Reply:
I think the idea is that it showcases the gem: I ran `rake doc` to generate the root readme (and an html file in `./site` which is in gitignore).

Reply:
The gem is literally called Double Doc, so… 😂